src/eko/output/struct.py

```diff
@@ -262,13 +262,16 @@ def __setitem__(self, q2: float, op: Optional[Operator], compress: bool = True):
             # at the moment, an implicit dependency on `tar` command has been
             # introduced -> dangerous for portability
             # since it's not raising any error, it is fine to run in any case:
-            # if the file is not there, nothing happens
-            subprocess.run(
-                f"tar -f {self.path.absolute()} --delete".split() + [info.name]
-            )
+            has_file = False
```

Comment:
in case you're doing for performances, at least remove both lines of the comment (not just the second one) and replace with the reason

otherwise in a few weeks we'll wonder why we are doing this

Reply:
if it's not for performances, then it's actually doing nothing...

Reply:
see [above](https://github.com/N3PDF/eko/pull/105#issuecomment-1190523057)